src/components/Loader/Person/Person.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../../types/Person';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => (
+  <td>
+    <Link
+      className={classNames('', {
```

Comment:
The 'className' attribute is assigned an empty string as the first argument in 'classNames'. You can remove the empty string if it's not necessary.